packages/apps/plugins/plugin-table/src/components/TableMain.tsx

```diff
@@ -11,12 +11,19 @@ import { ObjectTable, type ObjectTableProps } from './ObjectTable';
 
 const TableMain: FC<ObjectTableProps> = ({ table }) => {
   const containerRef = useRef<HTMLDivElement | null>(null);
+
   return (
     <Main.Content
       classNames={[baseSurface, 'fixed inset-inline-0 block-start-[--topbar-size] block-end-0 overflow-auto']}
       ref={containerRef}
     >
-      <ObjectTable table={table} stickyHeader role='grid' getScrollElement={() => containerRef.current} />
+      <ObjectTable
+        table={table}
+        key={table.id.toString()}
```

Comment:
I think this will always be a string so there shouldn't be a need for `.toString()`

Reply:
Typed as `Symbol` at the moment. Maybe I should just do `as string`?

Reply:
Ah, for some reason the type was coming through as `Symbol` but now it's string.